### PR TITLE
FIR IDE: Add quickfix for PRIVATE_SETTER_FOR_(OPEN|ABSTRACT)_PROPERTY

### PR DESCRIPTION
**Pull request by tgeng**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -26,7 +26,7 @@ kotlin.build.publishing.attempts=20
 
 #jpsBuild=true
 #cidrPluginsEnabled=true
-#idea.fir.plugin=true
+idea.fir.plugin=true
```
Comment (jsjeon) [inline gradle.properties:29]: I guess this shouldn't be enabled yet?

Comment (tgeng) [reply gradle.properties:29]: Ooops!!

File: idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt [hunk 2]
```diff
@@ -61,6 +61,7 @@ class RemoveModifierFix(
                 isRedundant = true
             )
         val removeOpenModifier = createRemoveModifierFromListOwnerPsiBasedFactory(KtTokens.OPEN_KEYWORD)
+        val removePrivateModifier: QuickFixesPsiBasedFactory<PsiElement> = createRemoveModifierFromListOwnerPsiBasedFactory(KtTokens.PRIVATE_KEYWORD)
```
Comment (jsjeon) [inline idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt:64]: Nit: explicit type for `val` seems not necessary (as the similar one above didn't specify that).

**Review by jsjeon — APPROVED** [review]

File: idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt (hunk 2 above)
Comment (tgeng) [reply idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt:64]: Let me change the line above to specify it so it's consistent with the ones above.

**Review by punzki — APPROVED** [review]

File: idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt (hunk 2 above)
Comment (jsjeon) [reply idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt:64]: Oh, everything except the above one specified the type, cool!

**Comment by darthorimar** [comment]
LGTM, thank you

**Pull request merged**